### PR TITLE
Task popup messages

### PR DESCRIPTION
**Pull request by m3c-ode**
## Content
Implemented notifications toaster using the react-hot-toast package.

## Related Issue
fixed

### CODE REVIEW

File: node_modules/.package-lock.json
Comment (akaka0039) [inline node_modules/.package-lock.json:1]: This file is not necessary, so can you delete this file ?

File: package-lock.json
Comment (akaka0039) [inline package-lock.json:1]: This file is not necessary, so can you delete this file ?

File: package.json
Comment (akaka0039) [inline package.json:1]: This file is not necessary, so can you delete this file ?

File: src/package-lock.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
-    "name": "app",
+    "name": "src",
```
Comment (akaka0039) [inline src/package-lock.json:2]: I think you did a command, which is 'npm run dev' in your local.
However it should do the command in docker container, which is 'app'.

File: src/resources/js/Components/Pages/Event/EventDetail.jsx [hunk 2]
```diff
@@ -461,6 +449,7 @@ const EventDetail = ({
                     </div>
                 </div>
             </div>
+            <Toaster />
```
Comment (akaka0039) [inline src/resources/js/Components/Pages/Event/EventDetail.jsx:452]: It looks so nice !!!
but I would like to display the message around the circle.
The hint is in [this document](https://react-hot-toast.com/docs/styling)

Screenshot 2023-08-13 at 10 23 16

File: node_modules/.package-lock.json
Comment (m3c-ode) [reply node_modules/.package-lock.json:1]: ok, I should be able to do this tomorrow (will be home then). Thank you!

File: src/package-lock.json (hunk 1 above)
Comment (m3c-ode) [reply src/package-lock.json:2]: I think it happened when I tried to fetch. I did not do this on purpose actually too. I'd like to hear more about this ;-)

File: src/resources/js/Components/Pages/Event/EventDetail.jsx (hunk 2 above)
Comment (m3c-ode) [reply src/resources/js/Components/Pages/Event/EventDetail.jsx:452]: Ok, I'll look into that as well!

**Review by akaka0039 — APPROVED** [review]

**Pull request merged**